.utils/deploy.sh

```diff
@@ -36,14 +36,14 @@ CSPSTATIC="\"content-security-policy\": \"default-src 'none'; "\
 # DO NOT REMOVE the locale from the newsletter form url. See #476.
 CSP="\"content-security-policy\": \"default-src 'none'; "\
 "base-uri 'self'; "\
-"connect-src 'self' https://blog.mozilla.org/addons/feed/ https://www.mozilla.org/en-US/newsletter/ https://www.google-analytics.com/; "\
+"connect-src 'self' https://blog.mozilla.org/addons/feed/ https://www.mozilla.org/en-US/newsletter/ https://*.google-analytics.com; "\
```

Comment:
Needed because GA4 makes requests to other subdomains, like `region1.google-analytics.com`. We do the same thing on AMO.